src/pudl/transform/eia860.py

```diff
@@ -161,6 +161,16 @@ def clean_ownership_eia860(raw_ownership_eia860: pd.DataFrame) -> pd.DataFrame:
     own_df["owner_country"] = own_df["owner_state"].map(state_to_country)
     own_df.loc[own_df.owner_state == "CN", "owner_state"] = pd.NA
 
+    # Spot fix NA generator_id. Might want to change this once we have the official 2022
+    # data not just early release.
+    constraints = (own_df["plant_id_eia"] == 62844) & (
+        own_df["report_date"].dt.year == 2022
+    )
+    if len(own_df[constraints]) > 1:
+        raise AssertionError("Too many records getting spot fixed.")
+
+    own_df.loc[constraints, "generator_id"] = "1"
+
```

Comment:
This is a bit of a hack-y way to get around an error that was throwing due to their being a null value in the primary key (i.e., `generator_id`) for just one row in this table. The plant doesn't show up for any other years in this table or the generators table. It does show up in the plants table, however, so I'm hoping it's just a quirk of the early_release. If you have a better suggestion of how to handle this I'd love to know. If, for some reason, this plant shows up in the generators table later on with a different generator id that would obviously not be good.

Reply:
Hmm, that's unfortunate! Does it show up in the EIA-860M data by chance?

It might be good to apply the spot-fix contingent on seeing an actual NA in the generator ID, so if it does get filled in later we're not clobbering it.  Or maybe we just have an assertion that fails if the ID shows up later, and we can then remove the hack.

Reply:
That's a good suggestion, I'll do that!